code/modules/paperwork/photocopier.dm

```diff
@@ -239,6 +244,27 @@
 	return p
 
 
+
+
+/// Upgraded photocopier, straight upgrade from the Normal Photocopier, made by Weyland-Yutani
+
+/obj/structure/machinery/photocopier/wyphotocopier
+	name = "photocopier"
+	icon = 'icons/obj/structures/machinery/library.dmi'
+	icon_state = "bigscannerpro"
+	desc = "A photocopier used for copying... you know, photos! Also useful for copying documents in paper. This specific model have been manufactured by Weyland-Yutani in a more modern and robust frame than the average photocopiers you see being sold by smaller companies, while also carrying some of the most advanced technologies on the area of paper-printing, such as bigger toner economy and much higher printing capabilities. All that makes it the favorite among consumers that need to print high amounts of paperwork on the performance of their daily duties."
+	idle_power_usage = 50
+	active_power_usage = 300
+	copies = 1 //how many copies to print!
+	toner = 180 //how much toner is left! woooooo~
+	maxcopies = 30 //how many copies can be copied at once- idea shamelessly stolen from bs12's copier!
+	animate_state = "bigscannerpro1" //what makes the animated icon flick when needed
```

Comment:
```suggestion
	copies = 1
	toner = 180
	maxcopies = 30
	animate_state = "bigscannerpro1"
```
As autodoc comments, your comments can be viewed from the parent's definition just by hovering over the variable in VSC.